### ISSUE TITLE
 Error setting up entry Nest Protect for nest_protect (since update 2023.5.0)

### ISSUE THREAD

**Issue by trunks403** [iMicknl/ha-nest-protect#226]
### The problem

Logger: homeassistant.config_entries
Source: custom_components/nest_protect/__init__.py:126
Integration: Nest Protect ([documentation](https://github.com/imicknl/ha-nest-protect), [issues](https://github.com/imicknl/ha-nest-protect/issues))
First occurred: 09:16:27 (1 occurrences)
Last logged: 09:16:27
Error setting up entry Nest Protect for nest_protect

Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 387, in async_setup
    result = await component.async_setup_entry(hass, self)
  File "/config/custom_components/nest_protect/__init__.py", line 126, in async_setup_entry
    hass.config_entries.async_setup_platforms(entry, PLATFORMS)
AttributeError: 'ConfigEntries' object has no attribute 'async_setup_platforms'

### What version of this integration (ha-nest-protect) has the issue?

V0.3.12

### What version of Home Assistant Core has the issue?

2023.5.0

### Device / Model

nest protect all sensors

### Diagnostics information

since the update to 2023.05.0 this interpretation is not working 

### Home Assistant log

<details><summary>Logs</summary>
```
Copy/paste any log here, between the starting and ending backticks (`)
```

</details>

**Comment by chrisdfw** [comment]
This one is workign for me

https://github.com/Cruguah/ha-nest-protect

**Comment by iMicknl** [comment]
Duplicate of https://github.com/iMicknl/ha-nest-protect/issues/224.

TLDR: make sure you don't use any fork and use  v0.3.12 (old login method) or v0.4.0b1 (cookie method).
